requirements.txt

```diff
@@ -8,3 +8,4 @@ pandas
 torch
 scipy
 tabulate
+pyperplan
```

Comment:
Is this installable via `pip`? I remember we had some issues before where we actually needed to install from source? I guess the checks pass so it's okay...

Reply:
yeah it is, right